packages/react/package.json

```diff
@@ -36,6 +46,8 @@
     ]
   },
   "devDependencies": {
+    "@aws-amplify/ui-core": "^0.0.1",
+    "@aws-amplify/ui-theme-base": "^0.0.1",
```

Comment:
Build tools like `microbundle`, `tsup`, etc. externalize `dependencies` and inline `devDependencies`.

This is helpful for using packages that we don't want to publish.

Reply:
Broader question: do we want to publish ui-theme-base? I noticed we publish @aws-amplify/ui which is kind of similar to what ui-theme-base is now. I don't have a strong opinion, just wanted to bring it up. Should we rename ui-theme-base to just ui?

Reply:
No, any cross-platform/framework dependencies (like the state machine, design tokens, or even specs (we ran into an issue with that!)) ideally lives in `@aws-amplify/ui-core`.

But you're right to ask about [`@aws-amplify/ui`](https://github.com/aws-amplify/amplify-js/tree/main/packages/amplify-ui).

`ui-core` would be a new package, but I don't think it makes sense to have.  `ui-core` is kinda our equivalent of `ui`, so you're right – maybe we rename & merge `ui-core` and `theme-base` into `ui`.

The only thing that I found weird is `aws-amplify` installs it by default:
https://github.com/aws-amplify/amplify-js/blob/da8ef238928fdcad7ab307350ffff8448eec5b26/packages/aws-amplify/package.json#L47

But I have no idea why, or see the need for it.

The more I type it out, the more it makes sense for `@aws-amplify/ui` to contain our core/shared assets and `@aws-amplify/ui-*` be framework-specific stuff.  `core` just seemed to be common convention in the ecosystem (e.g. `@babel/core`).

Let's queue this up for another cleanup task.